.gitignore

```diff
@@ -4,7 +4,10 @@
 
 # Remove Cargo.lock from gitignore if creating an executable, leave it for libraries
 # More information here https://doc.rust-lang.org/cargo/guide/cargo-toml-vs-cargo-lock.html
-# Cargo.lock
+Cargo.lock
```

Comment:
Cargo.lock should be checked out for executables.

Reply:
sorry I'm not sure I understand, should it be included or not?

Reply:
It should not be in `.gitignore`.

Reply:
```suggestion
```

Reply:
Fixed in commit #3cdd8e5